cassandra/src/test/scala/akka/stream/alpakka/cassandra/scaladsl/CassandraSourceSpec.scala

```diff
@@ -87,16 +89,16 @@ class CassandraSourceSpec extends WordSpec with ScalaFutures with BeforeAndAfter
       //#statement
 
       //#run-source
-      val rows = CassandraSource(stmt)
-        .runWith(Sink.seq)
+      val rows = CassandraSource(stmt).runWith(Sink.seq)
       //#run-source
 
       rows.futureValue.map(_.getInt("id")) must contain theSameElementsAs data
     }
 
     "support multiple materializations" in {
       val data = populate()
-      val stmt = new SimpleStatement("SELECT * FROM akka_stream_scala_test.test")
+      val stmt =
+        new SimpleStatement("SELECT * FROM akka_stream_scala_test.test")
```

Comment:
Is this a left over from the time where max column length was shorter? In one line it is 82 chars long.

Same unnecessary line break below.